### PR TITLE
Add linesAdded and linesRemoved to own interface of Commit

### PR DESCRIPTION
**Pull request by adelkahomolova**
Add linesAdded and linesRemoved to own interface of Commit so it can be used for Fat Commits Practice.

### CODE REVIEW

File: src/services/bitbucket/BitbucketService.ts [hunk 1]
```diff
@@ -99,41 +99,41 @@ export class BitbucketService implements IVCSService {
...
-        base: {
-          repo: {
-            url: val.destination.repository.links.html.href,
-            name: val.destination.repository.name,
-            id: val.destination.repository.uuid,
-            owner: {
-              login: owner,
-              id: ownerId,
-              url: ownerUrl,
+    const items = await Promise.all(
```
Comment (prokopsimek) [inline src/services/bitbucket/BitbucketService.ts]: not necessary

File: src/services/bitbucket/BitbucketService.ts [hunk 2]
```diff
@@ -346,6 +386,8 @@ export class BitbucketService implements IVCSService {
         email: this.extractEmailFromString(response.data.author.raw) || '',
         date: response.data.date,
       },
+      linesAdded: linesAdded,
+      linesRemoved: linesRemoved,
```
Comment (prokopsimek) [inline src/services/bitbucket/BitbucketService.ts]:
```
{
  lines: {
    added: number;
    deleted: number;
    changed: number;
  }
}
```

File: src/services/bitbucket/BitbucketService.ts (hunk 1 above)
Comment (adelkahomolova) [reply src/services/bitbucket/BitbucketService.ts]: @prokopsimek See lines 135 - 138

File: src/inspectors/common/ListGetterOptions.ts [hunk 3]
```diff
@@ -5,4 +5,5 @@ export interface ListGetterOptions<Filter = {}, SortOptions = {}> {
   filter?: Filter; // specific options to filter by (for example "state" for Github PR'ss)
   pagination?: PaginationParams; // general purpose pagination
   sort?: SortOptions; // specific sorting options
+  withDiffStat?: boolean;
```
[the remark this answers is not in the document]
Comment (adelkahomolova) [reply src/inspectors/common/ListGetterOptions.ts]: Are you sure? It's not filtering nor sort option.

[the remark this answers is not in the document]
Comment (prokopsimek) [reply src/inspectors/common/ListGetterOptions.ts]: You're true, but it's not a `ListGetterOptions` at all. Maybe it is misused.

**Pull request merged**